styleguide/components/11-base/fonts/_icons.scss

```diff
@@ -0,0 +1,141 @@
+@font-face {
+	font-family: "gent-icons";
+	src: url('../styleguide/fonts/gent-icons.eot');
```

Comment:
Add a unique version tag to these fonts :-) just to prevent caching ;-)

Reply:
Added.